CHANGES.rst

```diff
@@ -1,6 +1,21 @@
 Release notes
 =============
 
+v1.0.12 2025-12-02
+------------------
+
+* Include the 2-Yukawa structure factor model
+* Add Ruff pre-commit hooks for developers
+* Figures are cached between builds for CI speed up
+* Plugin model location docs updated
+
+v1.0.11 2025-08-19
+------------------
+
+* Integration of Ruff linting into CI
+* Code cleanup to fix linting errors
+* Fix map() bug in "sasfit_compare"
```

Comment:
In this case there are no user-facing changes. Maybe `* Maintenance release. No significant changes.`

Reply:
I'd argue `* Fix map() bug in "sasfit_compare"` is a user-facing change.

Reply:
It is in the "explore" directory, so it is not part of the public code base.